src/domain/common/port/DataFetcherPort.ts

```diff
@@ -0,0 +1,3 @@
+export default interface DataFetcherPort {
```

Comment:
You can use `export interface` as well. Also not default exports are more easy to support by IDE ;)

Reply:
But you don't have to*

Reply:
I will keep it as is to be consistent with other interfaces.